assets/css/style.css

```diff
@@ -104,6 +104,40 @@ html {
   scroll-behavior: smooth;
 }
 
+/* preloader */
```

Comment:
I am a but confused. when we are having a `style.css`, what is the need for a `preloader.css`?

Reply:
Actually, the preloader that you suggested had quite some lengthy CSS and as it was like a component that was reused for each page, so I personally thought of creating a separate file for the preloader, so that the style.css file remain clean and maintainable.

Reply:
@smaranjitghose I can do it with the only style.css if you don't like the idea. Please tell me the changes you want.